src/utils.ts

```diff
@@ -80,3 +81,25 @@ export function setEnvironmentVariables (envVars: Record<string, string> = {}) {
     process.env[key] = value;
   }
 }
+
+/**
+ * Check project's module type from an included package.json and return true
+ * if configured for ESM.
+ */
+export async function isEsmProject(projectPath?: string) {
```

Comment:
Hm. `projectPath` should never be empty under normal operation, no? Just curious why you went with it being optional here. If it's to handle the just-in-case scenario, perhaps we should log the error then? Would be useful for the time that it does happen :D